recipes/sqlalchemy_schemadisplay/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sqlalchemy_schemadisplay" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a9f26d77be9d92c9564d87cc17668fe141a816036c5f5d7c8cb053b253957e0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - pango >=1.40.14  # [not win]
+    - pydot
+    - python
+    - sqlalchemy
+
+test:
+  imports:
+    - sqlalchemy_schemadisplay
+
+about:
+  home: https://github.com/fschulze/sqlalchemy_schemadisplay
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: Turn SQLAlchemy DB Model into a graph
+  dev_url: https://github.com/fschulze/sqlalchemy_schemadisplay
+
+extra:
+  recipe-maintainers:
```

Comment:
@igorcandeia
Do you agree in co-maintain this recipe as well?

Reply:
Yes, I do.